### PR TITLE
MEN-3272: Make Raspberry Pi 4 a supported board.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: 03.Devices/03.Debian-family/docs.md [hunk 1]
```diff
@@ -142,11 +142,14 @@ MENDER_ARTIFACT_NAME=release-1 ./docker-mender-convert \
     --config <path to custom configuration file>
 ```
 
-<!--AUTOVERSION: "mender-convert/blob/%/configs"/ignore -->
+<!--AUTOVERSION: "mender-convert/blob/%/configs"/mender-convert -->
```
Comment (lluiscampos) [inline 03.Devices/03.Debian-family/docs.md:145]: :+1:

File: 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ Quickly and easily deploy your first over-the-air (OTA) software update with Men
 
 To follow this guide, you will need the following:
 
-* A [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b?target=_blank) or [B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus?target=_blank).
+* One of these Raspberry Pi models:
+  * [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/?target=_blank)
+  * [Raspberry Pi 3 Model B+](https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/?target=_blank).
```
Comment (lluiscampos) [inline 01.Getting-started/01.Quickstart-with-raspberry-pi/docs.md:16]: Nice addition :+1:

File: 04.Artifacts/10.Yocto-project/01.Building/docs.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ See [certificate troubleshooting](../../../troubleshooting/mender-client#certifi
 ### Starting from scratch
 
 Full details for building the Yocto project for your board are available at Mender Hub. The tested reference platforms for Mender are available at the following:
+* [Raspberry Pi 4 Model B](https://hub.mender.io/t/raspberry-pi-4-model-b/889)
 * [Raspberry Pi 3 Model B/B+](https://hub.mender.io/t/raspberry-pi-3-model-b-b/57)
```
Comment (lluiscampos) [inline 04.Artifacts/10.Yocto-project/01.Building/docs.md:63]: Just for consistency with other parts of the docs, I would list first Raspberry Pi 3 and then Rpi 4

File: 04.Artifacts/20.Provisioning-a-new-device/docs.md [hunk 4]
```diff
@@ -47,4 +47,4 @@ sudo dd if=<PATH-TO-IMAGE>.sdimg of=<DEVICE> bs=1M && sudo sync
...
 
-!!! &lt;DEVICE&gt; depends on where your SD card is placed. Normally this would be something like  `/dev/mmcblk0` or `/dev/sdb`.  If you are unsure how to find the correct device, the Raspberry PI Foundation provide some nice references that can help you for [Linux](https://www.raspberrypi.org/documentation/installation/installing-images/linux.md?target=_blank), [Mac OSX](https://www.raspberrypi.org/documentation/installation/installing-images/mac.md?target=_blank), [Windows](https://www.raspberrypi.org/documentation/installation/installing-images/windows.md?target=_blank).
+!!! &lt;DEVICE&gt; depends on where your SD card is placed. Normally this would be something like  `/dev/mmcblk0` or `/dev/sdb`.  If you are unsure how to find the correct device, the Raspberry Pi Foundation provide some nice references that can help you for [Linux](https://www.raspberrypi.org/documentation/installation/installing-images/linux.md?target=_blank), [Mac OSX](https://www.raspberrypi.org/documentation/installation/installing-images/mac.md?target=_blank), [Windows](https://www.raspberrypi.org/documentation/installation/installing-images/windows.md?target=_blank).
```
Comment (lluiscampos) [inline 04.Artifacts/20.Provisioning-a-new-device/docs.md:50]: Once we are on the tiny details, "the Raspberry Pi Foundation provide**s** some nice..."

File: 08.Downloads/docs.md [hunk 5]
```diff
@@ -22,12 +23,14 @@ Mender provides images based on the following distributions:
 !! manufacturer, or [contact us](mailto:contact@mender.io) if you have any
 !! questions on the Mender integration.
 
-| Board            | OS                              | Disk image                                                               | Storage size |
-|------------------|---------------------------------|--------------------------------------------------------------------------|--------------|
-| Raspberry Pi 3   | Raspbian Buster Lite 2020-02-05 | [raspbian-buster-lite-mender.img.xz][raspbian-buster-lite-mender.img.xz] | 8 GB         |
+| Board            | OS                              | Disk image                                                                                         | Storage size |
+|------------------|---------------------------------|----------------------------------------------------------------------------------------------------|--------------|
+| Raspberry Pi 3   | Raspbian Buster Lite 2020-02-05 | [raspbian-buster-lite-raspberrypi3-mender.img.xz][raspbian-buster-lite-raspberrypi3-mender.img.xz] | 8 GB         |
```
Comment (lluiscampos) [inline 08.Downloads/docs.md:28]: I think we should mention model B/B+

File: 04.Artifacts/20.Provisioning-a-new-device/docs.md (hunk 4 above)
Comment (kacf) [reply 04.Artifacts/20.Provisioning-a-new-device/docs.md:50]: Well spotted, will fix! Hehe, how long did it take you to spot the 'i'? 😀

**Comment by kacf** [comment]
Updated!

**Pull request merged**